nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpResponse.java

```diff
@@ -169,6 +174,7 @@ public void onTrigger(final ProcessContext context, final ProcessSession session
             return;
         }
 
+        session.getProvenanceReporter().send(flowFile, flowFile.getAttribute(HTTP_REQUEST_URI), stopWatch.getElapsed(TimeUnit.MILLISECONDS));
```

Comment:
@markap14 Request URI will be the URI requested and handled at HandleHttpRequest level. If I set a HandleHttpRequest and a HandleHttpResponse, then I send a GET request to, let's say, http://<nifi>:80/test, then the Request URI will be /test.
I definitely agree with you: we could add more details, but I was not sure about what we want to provide as information regarding the request that has been made initially. Should we use the IP address of the NiFi instance that received the request? Or information related to the client of the request?